### PR TITLE
better timezone handling

### PR DESCRIPTION
**Pull request by JarbasAI**
check if getting timezone for location failed, display would error and we get a speak message saying time is none

missed many common locations for me, added geopy and tzwhere to get timezone from any address

tzwhere is not working, requirements using fork for un-merged PR

### CODE REVIEW

**Comment by Mn0491** [comment]
@JarbasAI Just wondering, which locations did the time failed for you?

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 pytz==2017.2
 tzlocal==1.3
 astral==1.4
+geopy
+git+https://github.com/seahawk1986/pytzwhere.git@fix_install
```
Comment (Mn0491) [inline requirements.txt]: pip install fails right here for me

**Review by LearnedVector — COMMENTED** [review]
@JarbasAI a suggestion. Can you use the geopy library instead. Your pytzwhere library has a few apt dependencies, and we don't want to introduce that for a skill. with geo py you can do this...

```
import geopy from geocoder
g = geocoders.GoogleV3()
g.geocode("los angeles").address
// Los Angeles, CA, USA
```

File: requirements.txt (hunk 1 above)
Comment (Mn0491) [reply requirements.txt]: could you remove the pytzwhere from requirements.txt

**Review by LearnedVector — CHANGES_REQUESTED** [review]
request a change to requirements.txt. remove pytzwhere from it as it fails if apt-get packages are missing.

**Comment by Mn0491** [comment]
@JarbasAI can you see above

**Comment by penrods** [comment]
I've looked in to this a little bit.  I think https://pypi.python.org/pypi/timezonefinder/2.1.2 is a good library and built upon the open work in https://github.com/evansiroky/timezone-boundary-builder, but it comes at a cost of just under 100 MB of storage demands.  This seems a little heavy for a skill.

https://pypi.python.org/pypi/tzwhere is also big, using the same dataset I believe.

So to me the question is whether this should really live inside a skill or if this should be a service provided by a backend.  I lean towards the latter because the info is useful in other skills ("what is the weather like tomorrow afternoon in Stockholm?").

Perhaps this is a case where Mycroft AI shoulders the burden of keeping this sort of information up to date, likely via something like a Google Maps license?

**Pull request closed without merging**